### ISSUE TITLE
[Accessibility][Info & Relation -Flow Chart] : Narrator didn't announce anything when the narrator focus goes on the whole node.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2058]
**User Experience** 
The screen reader dependent user will not be able to know the presence of the node which will confuse the AT user.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE.
2. Select 'Design Flow' from left navigation.
3. Now, navigate to the flow chart.
4. Observe the narrator narration on the node name like "Send a response".

**Actual:** 
The narrator didn't announce anything when the focus lands on the node name.

**Expected:**
The narrator should announce the name of the node so that user will know the current position of the focus.

**Note:**
When the focus reaches on the node, the property pane gets appeared on the right side of window. The narrator should inform about the pane appearance.
Same issue found with the closing node. The narrator remains silent on the closing node.

**Observation:**
Same issue observed with NVDA.

**MAS Impacted:** [MAS1.3.1](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4})

**Attachment:**
[MAS1.3.1_Narrator didn't inform about the node name.zip](https://github.com/microsoft/BotFramework-Composer/files/4241231/MAS1.3.1_Narrator.didn.t.inform.about.the.node.name.zip)

**Comment by corinagum** [comment]
![Narrator didn't announce anything on node name](https://user-images.githubusercontent.com/14900841/75415456-883f7480-58e0-11ea-86b3-90e2d9d037c8.png)

.png uploaded for ease of access; note that .zip file also contains a video

**Comment by beyackle** [comment]
Back to working on this. I have a mostly-working solution, but it's still reading the wrong thing out sometimes and I'm figuring that out.

**Comment by ashish315** [comment]
As checked on below environment the issue is still repro.
**Test Environment**
Browser: Microsoft Edge Dev {Version 83.0.467.0 (Official build) dev (64-bit)}
Browser: Microsoft Edge Canary {Version 83.0.474.0 (Official build) canary (64-bit)}
OS build: 2004 (19588.1000)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Comment by corinagum** [comment]
I am still able to reproduce this problem, both on Narrator/Edge and NVDA/Chrome. However, I do see the aria-labels on the header and card node: 

![image](https://user-images.githubusercontent.com/14900841/78834203-0fcdd800-79a3-11ea-8b0c-67e7ac8f2a83.png)

![image](https://user-images.githubusercontent.com/14900841/78834394-5e7b7200-79a3-11ea-9542-225dc8a79ca1.png)

**Comment by corinagum** [comment]
@ashish315 do you have any insight on what the solution might be?

**Comment by ashish315** [comment]
@Corinagum May be their is some sync up issue due to which the screen reader focus remains on the More Option button when user press tab and the keyboard focus lands on the card due to which screen reader remains silent.

**Comment by beyackle** [comment]
I'm trying to repro this one again to return to work on it, and I can't even focus the cards inside the visual editor now. The entire editor gets read as "Visual editor - step editor - group" and then the Tab key won't move me anywhere, forward or backward. I'll look into that problem first.

**Comment by ashish315** [comment]
@beyackle Already highlighted this issue yesterday Bug ID: [2056](https://github.com/microsoft/BotFramework-Composer/issues/2056). I think you talking about this issue.

**Comment by beyackle** [comment]
That's a separate issue; see my comment on #2056

**Comment by beyackle** [comment]
That separate issue is already its own bug, too (#2021), so I'm going to call this one blocked on that. There's no way for focus to go onto a node in the first place until that one gets fixed.

**Comment by Amit8527510735** [comment]
#Nonbenchmark;
